index.html

```diff
@@ -10,7 +10,48 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header class="header__header">
+        <h1>Weather Report</h1>
+        <span>For the lovely city of 
+            <span id="headerCityName" class="header__city-name"></span></span>
```

Comment:
nit: Prefer matching the closing tags at the same level of indent as opening tags

```suggestion
        <span>For the lovely city of 
            <span id="headerCityName" class="header__city-name"></span>
        </span>
```